improver/cli/day_night_mask.py

```diff
@@ -0,0 +1,62 @@
+#!/usr/bin/env python
```

Comment:
I think that it might make sense to rename this CLI (and associated test directories) to be `apply_night_mask`, rather than `day_night_mask`, as the output is the same as the input but with night values masked.

Reply:
done, and changed input directory to apply-night-mask